client/config/environment.js

```diff
@@ -0,0 +1,64 @@
+/* jshint node: true */
+
+module.exports = function(environment) {
+  var ENV = {
+    modulePrefix: 'vilive',
+    environment: environment,
+    baseURL: '/',
+    locationType: 'auto',
+    EmberENV: {
+      FEATURES: {
+        // Here you can enable experimental features on an ember canary build
+        // e.g. 'with-controller': true
+      }
+    },
+
+    APP: {
+      // Here you can pass flags/options to your application instance
+      // when it is created
+    }
+  };
+
+  ENV.yggdrasil = {
+    endpoint: 'http://localhost:4000/api/auth/login'
```

Comment:
this probably can move to APP don't know yet, because the JSONAPI adapter for ember data will need to know the base url for the server as well for all it's requests.